examples/interactivity/feature-click.html

```diff
@@ -75,24 +75,20 @@ <h2></h2>
     });
 
     const source = new carto.source.Dataset(`
-      ne_adm0_europe
+      ne_10m_populated_places_simple
```

Comment:
Any special reason for this change? I find easier to click in countries (big) rather than cities (small points)

Reply:
I just decided to use points for click, enter/leave for polygons and hover for lines. @makella will update the examples in the future if required